### ISSUE TITLE
Override link behaviour of html anchors included in manifests

### ISSUE THREAD

**Issue by aeschylus** [ProjectMirador/mirador#1944]
Sometimes links provided in the html content of manifest metadata open in the current tab, and sometimes they open in another tab. We should override this behaviour so all supplemental information opens in a new browser tab.

**Comment by mejackreed** [comment]
I'm conflicted here. Some part of me seems to think it is wrong to modify the internal metadata from a manifest to coerce a desired behavior, while at the same time I recognize the user experience problem this presents.

**Comment by ggeisler** [comment]
@mejackreed Is it really _modifying_ the manifest metadata, though? Technically, it is (because we're adding link behavior not specified in the link), but in terms of the manifest creator's _intention_ for the URL? 

For us to be modifying the manifest creator's intention for the URL, we'd have to assume that the manifest provider consciously chose **not** to include a `target="_blank"` attribute in the metadata URL. That is, we'd have to assume that all manifest providers intentionally indicate their desired link behavior in manifest metadata URLs (by including or omitting a `target` attribute). But is that a realistic assumption, given that IIIF manifests aren't intended to be written with a specific consuming client in mind?

In other words, as a manifest creator can I really be expected to specify whether I want my link to open in a new tab or not, when I don't know what sort of client or application it might be consumed by? 

I understand your point, I think, but to me it feels reasonable for consuming applications to make a decision on how the link should behave, to offer the best user experience for working with the manifest object, based on the particular application.

**Comment by charbugs** [comment]
@aeschylus Could you give an example for a manifest where links open in new tab? I'm wondering why this happens because the html rule set we currently use doesn't allow the target attribut.
